packages/itwinui-css/src/listbox/listbox.scss

```diff
@@ -0,0 +1,22 @@
+// Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+// See LICENSE.md in the project root for license terms and full copyright notice.
+
+@mixin iui-listbox-wrapper {
+  --iui-menu-listbox-height: calc((var(--iui-component-height) - 1px) * 4.5);
```

Comment:
why not just set `height`?

Reply:
Trying to refresh my memory on this one, it's been a while.

I think I set the height on the `class="iui-list" role="listbox"` to avoid inconsistent heights if the listboxes contained a different amount of items:
![Screenshot 2023-04-06 at 4 43 29 PM](https://user-images.githubusercontent.com/849817/230490100-6587bb9e-4022-4494-a634-665c700531c1.png)

And if `flex: 1` was applied but there was other stuff surrounding it:
![Screenshot 2023-04-06 at 4 44 28 PM](https://user-images.githubusercontent.com/849817/230490271-9d1537de-560f-4707-9eaf-7b6dc712f6a6.png)

Pink background applied to `.iui-listbox-wrapper`.

Reply:
Let me rephrase the question. Why not set `height: calc((var(--iui-component-height) - 1px) * 4.5)` directly on the listbox rather than set a CSS variable here first and use it in the listbox?

I don't see how having different number of items affects it.

Reply:
Height is now being set directly in listbox and not through the use of a variable

Reply:
yes, thank you!